.github/workflows/nightly.yml

```diff
@@ -0,0 +1,125 @@
+name: Nightly
+
+on:
+  schedule:
+    - cron: 0 0 * * *
+
+
+jobs:
+  generate-proving-key:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: dev
+
+      - name: Use Node.js 16
+        uses: actions/setup-node@v4
+        with:
+          node-version: 16
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install --yes \
+            build-essential \
+            libgmp-dev \
+            libsodium-dev \
+            nasm \
+            nlohmann-json3-dev
```

Comment:
nlohmann-json3-dev is not needed anymore

Reply:
It appears that removing nlohmann-json3-dev leads to a compilation failure. Here's the error we encounter:

```
main.cpp:9:10: fatal error: nlohmann/json.hpp: No such file or directory
    9 | #include <nlohmann/json.hpp>
      |          ^~~~~~~~~~~~~~~~~~~
compilation terminated.
```

Reply:
mm I want to do some checks on a fresh VM actually. Looks like rapidnsark does not need this, but zkey-manager does, however where does it use it? is it here: https://github.com/privacy-scaling-explorations/zkey-manager/blob/master/ts/compile.ts#L102?

Reply:
It's needed when computing witness with c++: https://docs.circom.io/getting-started/computing-the-witness/